TECHNOLOGY_CHOICES.md

```diff
@@ -0,0 +1,52 @@
+# Technology Choices
+## Checks for determining database
+- Database should be able to store XML data
+- Database should be relational, because we need relations between files/pieces of files/models
+- Database must be provided with the solutions, for example part of a Docker image
+- Support must be available, in case of trouble.
+- Database must be open-source, because the whole solution must be open-source
+
+## PostgreSQL
+Homepage: https://www.postgresql.org/
+### Pros
+- Open Source, relational database
+- PostgreSQL does have XML functions for handling XML files (XMLTABLE)
+- XMLTABLE turns XML into a relational table format
+- Good support available
+- [Docker image](https://hub.docker.com/_/postgres) available
+
+### Cons
+- Not a native XML database
+
+## CouchDB
```

Comment:
What happend to Neo4j?

Reply:
Neo4J was one of the database I wanted to do, but I didn't do it. It can be removed :) (Which I will do)

Reply:
Their are reasons to move away from Neo4j, these can be noted.

Reply:
Keeping Neo4j out for now is fine for me.